scripts/deploy/deploy.sh

```diff
@@ -1,28 +1,13 @@
-SOMERVILLE_PROD_HEROKU_APP_NAME=$1
-DEMO_HEROKU_APP_NAME=$2
-
-if [ -z "$SOMERVILLE_PROD_HEROKU_APP_NAME" ]; then
-  echo "🚨  🚨  🚨  Please supply a Somerville production Heroku app name."; exit 0;
-fi
-
-if [ -z "$DEMO_HEROKU_APP_NAME" ]; then
-  echo "🚨  🚨  🚨  Please supply a demo Heroku app name."; exit 0;
-fi
-
-# Deploy to Somerville production app and migrate
-echo "🚢  🚢  🚢  Deploying code to the Somerville production app.";
-git push heroku master
-echo;
-
-echo "⚙  ⚙  ⚙  Migrating the database.";
-heroku run rake db:migrate --app "$SOMERVILLE_PROD_HEROKU_APP_NAME"
-echo;
-
-# Deploy to demo app and migrate
-echo "🚢  🚢  🚢  Deploying code to the demo Heroku app.";
-git push heroku-demo master
-echo;
-
-echo "⚙  ⚙  ⚙  Migrating the database.";
-heroku run rake db:migrate --app "$DEMO_HEROKU_APP_NAME"
-echo;
+# Pass in the name of all the git remotes to deploy to as arguments
+
+# Example: ./scripts/deploy/deploy.sh somerville new-bedford heroku-demo
+
+for remote in "$@"
+do
+    echo "🚢  🚢  🚢  Deploying code to $remote.";
+    git push $remote master
+    echo;
+    echo "⚙  ⚙  ⚙  Migrating the database for $remote.";
+    heroku run rake db:migrate --remote $remote
```

Comment:
@alexsoble I think my local git and Heroku CLI setup might be a bit different.  This seems great, and a good opportunity to sync these 👍 

To check my understanding, this means you have:
```
$ git remote -v
somerville 	https://git.heroku.com/somerville-teacher-tool.git (fetch)
somerville	https://git.heroku.com/somerville-teacher-tool.git (push)
new-bedford 	https://git.heroku.com/new-bedford-insights.git (fetch)
new-bedford	https://git.heroku.com/new-bedford-insights.git (push)
```

and then since you're using `--remote` instead of `--app`, you use the same `somerville` and `new-bedford` git remote names when running heroku commands.

Am I following?

Reply:
Yup! So basically we need to do two things, push code to github and run a heroku command. 

The github push requires a remote. The heroku command requires either a `--remote` flag, or `--app` flag. Since we could get them both to do the right thing using the git remote as a single argument, I went with that.

Reply:
My mental model is that you should be able to set the `<name>` values for your remotes locally to whatever you like, say somerville = `foo` and new bedford = `bar`, and the deploy script should work. Does that match up to what you're thinking?

Reply:
Thanks for teaching me!  This seems great and I updated my setup to match. 👍

Reply:
@kevinrobinson How did you update your setup?